bin/tests/it/cmd_line_env.rs

```diff
@@ -104,11 +104,10 @@ fn test_list_config_from_conf() -> io::Result<()> {
         .arg("-l")
         .unwrap();
     assert!(output.status.success());
-    let stdout = from_utf8(&output.stdout).unwrap();
+    let stdout = from_utf8(&output.stderr).unwrap();
```

Comment:
something changed in testing testing framework, output is captured in stderr instead of stdout which is empty.